### PR TITLE
[JA] Add `Beatmap/Play_time` and 5 other articles

### PR DESCRIPTION
**Pull request by yuzupon1133**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

`Beatmap/Play_time`\
`Beatmap/Countdown`
`Beatmap/Break`\
`Beatmap/Song_complation`\
`Beatmap/Online_offset`\
`Beatmap/Star_rating`

### CODE REVIEW

File: wiki/Beatmap/Countdown/ja.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - timer
+---
+
+# カウントダウン
+
+*トーナメント用のカウントダウンタイマーは[トーナメント管理コマンド](/wiki/osu!tourney/Tournament_management_commands)を参照してください。*\
+*カウントダウンに関する規則は[ランキング基準のガイドライン](/wiki/Ranking_Criteria#guidelines)を参照してください。*\
+*その他の用法については、[オフセット（曖昧さ回避）](/wiki/Disambiguation/Offset)を参照してください。*
+
+カウントダウンが有効な[ビートマップ](/wiki/Beatmap)は、最初のオブジェクトが表示される前に始まるまでのカウントが表示されます。これは、突然始まる曲に便利ですが、ほとんどのビートマップではこれを有効にする必要はありません。カウントダウンは[song setup](/wiki/Client/Beatmap_editor/Song_Setup)のDesignタブからオン/オフを切り替えることができますが、osu!taikoビートマップでは許可されていません。
```
Comment (TPGPL) [inline wiki/Beatmap/Countdown/ja.md:13]:
```suggestion
カウントダウンが有効な[ビートマップ](/wiki/Beatmap)は、最初のオブジェクトが表示される前に始まるまでのカウントが表示されます。これは、突然始まる曲に便利ですが、ほとんどのビートマップではこれを有効にする必要はありません。カウントダウンは[song setup](/wiki/Client/Beatmap_editor/Song_Setup)のDesignタブからオン/オフを切り替えることができますが、[osu!taiko](/wiki/Game_mode/osu!taiko)ビートマップでは許可されていません。
```

File: wiki/Beatmap/Countdown/ja.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+カウントダウンアニメーションは`Countdown Speed`(`Half`, `Normal`, `Double`)と`Countdown Offset`で制御できます。`Countdown Speed`は、カウントする速度が左から0.5倍、1倍、2倍になります。`Countdown Offset`は最初の[ヒットオブジェクト](/wiki/Hit_object)の何[拍](/wiki/Beatmapping/Beats_per_minute)前に開始するかを設定します。例えば、オフセット2に設定したカウントダウンでは、カウントダウンの開始が通常より2拍早くなります。イントロ付きのマップの場合、スキップするとカウントダウン前に直接移動します。
+
+カウントダウンアニメーションはスキンで変更することができます。
```
Comment (TPGPL) [inline wiki/Beatmap/Countdown/ja.md:17]: missing link

for ref:

`The countdown animation can be [skinned](/wiki/Skinning/Interface#countdown).`

File: wiki/Beatmap/Play_time/ja.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+stub: true
+tags:
+  - length
+  - playtime
+  - total length
+---
+
+# プレイ時間
+
+**プレイ時間**は、プレイしたビートマップの合計の時間です。これには、曲の開始までの時間と休憩時間も含まれます。これは通常、ビートマップの曲の長さと比例しています。
```
Comment (TPGPL) [inline wiki/Beatmap/Play_time/ja.md:11]: missing link

for ref:

`**Play time** is the total duration of a [beatmap](/wiki/Beatmap). This includes the lead-in time and the break time(s). This usually correlates with the beatmap's song length.`

File: wiki/Beatmap/Song_compilation/ja.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+stub: true
+tags:
+  - song compilations
+---
+
+# Song compilation
+
+**Song compilation**はいろいろな曲を一つにまとめたもので、通常はジャンルやアーティスト、その他のテーマに沿ってまとめられます。
+
+Song compilationのビートマップは、曲をまとめたものを1つのオーディオファイルにしています。通常は、それぞれの曲のコーラスや小節などの一部分を使います。フォルダ内に複数の曲ファイルを含むビートマップと混同しないでください。
```
Comment (TPGPL) [inline wiki/Beatmap/Song_compilation/ja.md:11]: missing link

for ref

`[Beatmaps](/wiki/Beatmap) of song compilations use a single audio file that contains the whole collection of songs, usually including only a single chorus or verse of each. This is not to be confused with a beatmap that includes multiple song files in its folder.`

File: wiki/Beatmap/Star_rating/ja.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - SR
+  - stars
+---
+
+# 難易度
+
+**難易度** (Star rating, SR)は、osu!によって自動的に計算されるビートマップの難易度を表します。難易度は、パフォーマンスポイントに大きな影響を与えます。難易度は曲の選択画面とビートマップ情報ページで確認できます。
```
Comment (TPGPL) [inline wiki/Beatmap/Star_rating/ja.md:10]: many links missing

for ref:

`**Star rating** (***SR***) describes a [beatmap](/wiki/Beatmap)'s difficulty, which is automatically calculated by the osu! client. Star rating plays a large role in how [performance points](/wiki/Performance_points) are awarded. This number can be seen on the [song selection screen](/wiki/Client/Interface#song-select) and on a beatmap's info page.`

**Review by TPGPL — DISMISSED** [review]

File: wiki/Beatmap/Online_offset/ja.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  - online song offset
+---
+
+# オンラインオフセット
+
+*その他の用法については、[オフセット（曖昧さ回避）](/wiki/Disambiguation/Offset)を参照してください。*
+
+**オンラインオフセット**は、[ビートマップ](/wiki/Beatmap)が[Ranked](/wiki/Beatmap/Category#ranked)になった後にビートマップに適用できるオフセットです。これは通常、ビートマップが曲のタイミングに合っていない場合に行われます。オンラインオフセットは、プレイヤーのグローバルオフセットとローカルオフセットの上に適用されます。
+
+これは、[ノミネーションアシスタントチーム](/wiki/People/The_Team/Nomination_Assessment_Team)(NAT)が設定でき、[ビートマップノミネーター](/wiki/People/The_Team/Beatmap_Nominators)(BN)や[開発者](/wiki/People/The_Team/Global_Moderation_Team)など、一部の昇格したユーザーからも見ることができます。また、オンラインオフセットは、Webサイトでは通常のユーザーからは見ることはできませんが、ゲーム画面では表示されます。
```
Comment (TPGPL) [inline wiki/Beatmap/Online_offset/ja.md:12]:
```suggestion
これは、[ノミネーションアシスタントチーム](/wiki/People/The_Team/Nomination_Assessment_Team)(NAT)が設定でき、[ビートマップノミネーター](/wiki/People/The_Team/Beatmap_Nominators)(BN)や[開発者](/wiki/People/The_Team/Global_Moderation_Team)など、一部の昇格したユーザーからも見ることができます。また、オンラインオフセットは、Webサイトでは通常のユーザーからは見ることはできませんが、ゲーム画面では表示されます。
```

**Comment by yuzupon1133** [comment]
Fixed😅

**Review by peppy — APPROVED** [review]

**Pull request merged**